EIPS/eip-draft_location-metadata-standard.md

```diff
@@ -0,0 +1,189 @@
+---
+eip: <to be assigned>
+title: Location Metadata Standard in ERC-721 Tokens
+author:
+Ricardo Chacon (@RicardoChacon)
+Isabella Costanza (@isabellacostanza)
+Gabriela Garro (@gabygarro)
+James Jose (@jamesjosegdsey)
+David Mendes (@de-mendes)
+discussions-to: https://ethereum-magicians.org/t/request-for-feedback-standardizing-location-metadata-and-other-metadata-in-erc-721-tokens/5985
+status: Draft
+type: Informational
+created: 2021-05-04
+requires: 721
+---
+
+## Simple Summary
+
+Standard for location metadata in the ERC-721 tokens. Tokens wishing to include physical location metadata should follow the same standard.
+
+## Abstract
+
+This standard specifies how location metadata should be represented in an ERC-721 token. It serves as an extension of the ERC-721 Metadata Standard as it adds a "location" field to the JSON schema. This location metadata can be represented with either the physical address, geolocation coordinates or both for maximum clarity. 
+
+## Motivation
+
+This standard will promote composability and cohesiveness of tokens across across platforms and systems. Use cases which rely on location metadata or that would benefit from implementing it often find themselves in the position of structuring this data in whichever way serves their specific purpose. However, representing location data is not a trivial endeavor and too many implementations exist since each one is use case specific, even though they are mostly similar. Therefore, we have leveraged existing ISO and geolocation standards to develop a metadata structure that can be applied globally.
```

Comment:
```suggestion
This standard will promote composability and cohesiveness of tokens across platforms and systems. Use cases which rely on location metadata or that would benefit from implementing it often find themselves in the position of structuring this data in whichever way serves their specific purpose. However, representing location data is not a trivial endeavor and too many implementations exist since each one is use case specific, even though they are mostly similar. Therefore, we have leveraged existing ISO and geolocation standards to develop a metadata structure that can be applied globally.
```

Reply:
Fixed